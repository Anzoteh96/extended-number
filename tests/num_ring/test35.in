-758970137572207001705428960394458964466914789135122444472353599427058806087420443234993496590204762531013
-475400332897002041493216392496085765830324219830300104144215507309374019539717937606592904848040944128488
329613885029926215672649780220021960420521056305508455922473494527072061745738297985475597347892669459858
