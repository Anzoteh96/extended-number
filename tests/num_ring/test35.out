Enter the number base b:
Enter two numbers n and m:
b is 758970137572207001705428960394458964466914789135122444472353599427058806087420443234993496590204762531013
n mod b is 283569804675204960212212567898373198636590569304822340328138092117684786547702505628400591742163818402525
m mod b is 329613885029926215672649780220021960420521056305508455922473494527072061745738297985475597347892669459858
n + m is 613183689705131175884862348118395159057111625610330796250611586644756848293440803613876189090056487862383
n - m is 712926057217485746244991748072810202682984302134436328878018197017671530889384650877918490984475911473680
n * m is 401646624965282213447362501424209562448491458963926124509461983977116601802979311385301644249021785214525
n ** m is 286328199174162019241691477788484169534104918879311840641579260313424210456226726563943763890665134373630
